data/json/items/gun/32.json

```diff
@@ -55,7 +55,7 @@
     "description": "The Skorpion Vz. 61 is a Czechoslovak submachine gun from the 1950s, chambered in .32 ACP.",
     "weight": "1300 g",
     "volume": "788 ml",
-    "longest_side": "1116 mm",
+    "longest_side": "270 mm",
```

Comment:
```suggestion
    "longest_side": "298mm",
```
We're supposed to use the longest length available, which is the Feret diameter.

![image](https://user-images.githubusercontent.com/51099123/84583867-fa729100-adcb-11ea-9901-73e4ee966290.png)